<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1211</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="LED_Panels">
     <item>
      <layout class="QHBoxLayout" name="LED_Panel_left">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="LED_A01">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A03">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A04">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A05">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A06">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A07">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A08">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A09">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_A16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="LED_Panel_right">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="LED_B01">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B03">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B04">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B05">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B06">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B07">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B08">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B09">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LED_B16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="IndicatorControl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>290</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Indicator Control</string>
    </property>
    <widget class="QPushButton" name="button_LEFT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_WARN">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>WARN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_RIGHT">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_DOORLOCK">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>65</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DOOR LOCK</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ControlPanel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>290</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QPushButton" name="button_ONOFF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>ON / OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_EXIT">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_CLOCK">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CLOCK</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ClockControl">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Clock Control</string>
    </property>
    <widget class="QLabel" name="label_SPEED_DISPLAY">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_SPEED">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>180</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="textinput_INTERVAL">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MS">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ms clock</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SoundControl">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Sound Control</string>
    </property>
    <widget class="QSlider" name="slider_SOUND">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>180</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="SoundsSelector">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Sound Selector</string>
     </property>
     <widget class="QRadioButton" name="radioButton_S1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>S1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_S2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>S2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_S3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>S3</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_VOLUME">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AnimationControl">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>290</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Animation Control</string>
    </property>
    <widget class="QLabel" name="label_LEFT">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_LEFT">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_RIGHT">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_WARN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>75</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_RIGHT">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>75</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>WARN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
